templates/artist/artist_detail.html

```diff
@@ -225,12 +230,15 @@ <h1 class="header-font">{{ artist.name }}</h1>
     <script src="https://checkout.stripe.com/checkout.js"></script>
     <script type="text/javascript" src="{% static "js/vendor/smooth-scroll.js" %}"></script>
     <script type="text/javascript">
+        var perdiem_fee = {{ PERDIEM_FEE }};
+        var artist_name = '{{ artist.name|escapejs }}';
+        {% if campaign %}
         var doughnutData = [
```

Comment:
Sweet. Now just indent lines 236-279 so that it's obvious that this JS only runs if there is a campaign.